jina/peapods/runtimes/asyncio/rest/models.py

```diff
@@ -24,12 +24,16 @@ def build_model_from_pb(name: str, pb_model: Callable):
     :return: Model.
     """
     from google.protobuf.json_format import MessageToDict
-
     dp = MessageToDict(pb_model(), including_default_value_fields=True)
-
     all_fields = {k: (name if k in ('chunks', 'matches') else type(v), Field(default=v)) for k, v in dp.items()}
     if pb_model == QueryLangProto:
         all_fields['parameters'] = (Dict, Field(default={}))
+    if pb_model == DocumentProto:
+        # these fields are defined as oneof in the jina.proto
+        # therefore, they are not instantiated when using pb_model()
+        all_fields['buffer'] = (str, Field()) #base64 encoded
```

Comment:
Any changes in proto would need a change here. This method should be generic & independent on proto level fields. I would fix this in a different PR.